src/esssans/types.py

```diff
@@ -82,6 +113,9 @@ class TransmissionRun(sciline.Scope[RunType, int], int):
 CorrectForGravity = NewType('CorrectForGravity', bool)
 """Whether to correct for gravity when computing wavelength and Q."""
 
+FinalDims = NewType('FinalDims', Sequence[str])
+"""Final dimensions of IofQ"""
```

Comment:
This needs more detail, it is completely unclear what valid options are, and what the point is.

Reply:
Changed to `DimsToKeep` and added a better description.